### PR TITLE
Feature/active filters

### PR DESCRIPTION
**Pull request by arbreurkes**
Show which filters are active in node/leaf info list.

### CODE REVIEW

**Comment by ghost** [comment]
No remarks regarding code quality.

File: src/main/java/application/controllers/GraphController.java [hunk 1]
```diff
@@ -376,7 +375,14 @@ public Image takeSnapshot() {
...
 
-        WritableImage image = new WritableImage(pref, height);
+        WritableImage image;
+
+        if (graph.getModel().getAllCells().isEmpty()) {
+            image = new WritableImage(1, 1);
+        } else {
+            image = new WritableImage(pref, height);
+        }
+
```
Comment (dvantetering) [inline src/main/java/application/controllers/GraphController.java:384]: This isn't correct -> please discuss with Niek

Comment (arbreurkes) [inline src/main/java/application/controllers/GraphController.java:384]: It is correct.

**Comment by tieswesterborg** [comment]
Ready to merge.

File: src/main/java/application/controllers/GraphController.java (hunk 1 above)
Comment (tieswesterborg) [inline src/main/java/application/controllers/GraphController.java:384]: Agree, this is correct

**Comment by tieswesterborg** [comment]
Travis wont fail. But lets wait to be sure.

**Pull request merged**